cli/cmd/volume.go

```diff
@@ -64,8 +69,11 @@ func init() {
 	// Volume Delete
 	volumeCmd.AddCommand(volumeDeleteCmd)
 
-	volumeCmd.AddCommand(volumeGetCmd)
+    //Volume Set
+	volumeSetCmd.Flags().StringVarP(&flagSetCmdData, "data", "d", "", "Volume Options")
```

Comment:
Why this is required?

Reply:
um, I had named the flag as 'option' earlier, rectified and renamed : '--data'

Reply:
Both of them are not required. Do not use any flags for volume set. Just accept all as arguments.

    $./glustercli volume set testvol afr.eager-lock on foo bar